### ISSUE TITLE
Section on social events

### ISSUE THREAD

**Issue by Oloni** [numfocus/DISCOVER-Cookbook#21]
Hi there,
While social events are implicitly mentioned, it would be great to have a dedicated section on how to make them inclusive & diverse, covering the type of accessible social events that could be offered (e.g. mobility impaired folk should be able to participate, so a long walk may be problematic) along with guidelines on how to make everyone feel welcome (e.g. don't make it alcohol centric, make everyone aware of a code of conduct etc.). Happy to write a draft, I haven't found any existing guidelines specific to social events.
Leonie

**Comment by Dr-G** [comment]
I can help with the content

**Comment by KameshSingh116** [comment]
Yes i can help[[ with the content and also outreach of the events.

**Comment by ChehakTrehan** [comment]
Hii @Oloni I added detailed section of social events. Please review my PR.
Thanks
Chehak
